Replace BNDW with VAGF (#346)
teamniteo/handbook: 5_People/personal-finances.md

We're looking to invest in the lowest cost funds by a large and reputable fund. Niteo, Dejan, and Nejc all invest in [Vanguard's](https://en.wikipedia.org/wiki/The_Vanguard_Group) funds:

> - VWCE for the global stock market (listed as VWCE in EUR)

Reviewer: ```suggestion
- VWCE (EUR) or VWRA (USD) for the global stock market
```

> - VAGF for the global bond market

Reviewer: ```suggestion
- VAGF (EUR) or BNDW (USD) for the global bond market
```
